backend/regalloc/regalloc_ls_state.ml

```diff
@@ -25,7 +25,7 @@ let[@inline] make ~stack_slots ~last_used =
   { interval_dll; active; stack_slots; instruction_id }
 
 type class_interval_array =
-  { elements : Interval.t array;
+  { mutable elements : Interval.t array;
```

Comment:
I am not against the changes in
this file (far from it), but out of
curiosity why did you have to
do this change?

Reply:
The current stamp is no longer the total reg count, so I deleted `Reg.total_regs` and did this instead